### PR TITLE
Fix perf:perftool 

### PR DESCRIPTION
**Pull request by PraveenPenguin**
as if command failed test error out and next block of test not at able to execute so adding
igonre_status=True in process.run

without patch:

19:05:59 INFO    : Running: /usr/bin/avocado run  avocado-misc-tests/perf/perftool.py --force-job-id 1a69a2ba6a8609322110aa064d6f895551fa1cad
JOB ID     : 1a69a2ba6a8609322110aa064d6f895551fa1cad
JOB LOG    : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.06-1a69a2b/job.log
 (1/1) avocado-misc-tests/perf/perftool.py:Perftool.test: ERROR: Command 'perf test' failed.\nstdout: ''\nstderr: " 1: vmlinux symtab matches kallsyms            : Skip\n 2: Detect openat syscall event                : Ok\n 3: Detect openat syscall event on all cpus    : Ok\n 4: Read samples using the mmap interface     ... (29.57 s)
RESULTS    : PASS 0 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 30.31 s
JOB HTML   : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.06-1a69a2b/results.html
19:06:31 INFO    :
19:06:32 INFO    : Summary of test results can be found below:
TestSuite                                                                   TestRun    Summary

host_perf1_perf1                                                            Run        Successfully executed
/home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.06-1a69a2b/job.log
1

with patch:

19:08:07 INFO    : Check for environment
19:08:07 INFO    : Creating temporary mux dir
19:08:09 INFO    :
19:08:09 INFO    : Running Host Tests Suite perf1_perf1
19:08:09 INFO    : Running: /usr/bin/avocado run  avocado-misc-tests/perf/perftool.py --force-job-id ff469469fbdee1d64d641869d0fb24a4c6487825
JOB ID     : ff469469fbdee1d64d641869d0fb24a4c6487825
JOB LOG    : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.08-ff46946/job.log
 (1/1) avocado-misc-tests/perf/perftool.py:Perftool.test: FAIL: 123 Test failed (472.16 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 472.78 s
JOB HTML   : /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.08-ff46946/results.html
19:16:04 INFO    :
19:16:04 INFO    : Summary of test results can be found below:
TestSuite                                                                   TestRun    Summary

host_perf1_perf1                                                            Run        Successfully executed
/home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/results/job-2019-07-23T19.08-ff46946/job.log
19:16:04 INFO    : Removing temporary mux dir

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@narasimhan-v can you please check this PR , as it blocking our CR

File: perf/perftool.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def test(self):
         '''
         count = 0
         # Built in perf test
-        for string in process.run("perf test").stderr.splitlines():
+        for string in process.run("perf test", ignore_status=True).stderr.splitlines():
```
Comment (narasimhan-v) [inline perf/perftool.py]: @PraveenPenguin 
1. why are you splitting the lines multiple times ?
2. Are we fine in failing fast here ? If so, the for loop is unnecessary

Comment (PraveenPenguin) [reply perf/perftool.py]: @narasimhan-v no it takes time to run test , and it goes one by one test case as example this is output
```
11:26:41 DEBUG| [stderr]  1: vmlinux symtab matches kallsyms            : Skip
11:26:41 DEBUG| [stderr]  2: Detect openat syscall event                : Ok
11:26:41 DEBUG| [stderr]  3: Detect openat syscall event on all cpus    : Ok
11:26:41 DEBUG| [stderr]  4: Read samples using the mmap interface      : Ok
11:26:41 DEBUG| [stderr]  5: Test data source output                    : Ok
11:26:41 DEBUG| [stderr]  6: Parse event definition strings             : Ok
11:26:41 DEBUG| [stderr]  7: Simple expression parser                   : Ok
11:26:41 DEBUG| [stderr]  8: PERF_RECORD_* events & perf_sample fields  : Ok
11:26:41 DEBUG| [stderr]  9: Parse perf pmu format                      : Ok
11:26:41 DEBUG| [stderr] 10: DSO data read                              : Ok
11:26:41 DEBUG| [stderr] 11: DSO data cache                             : Ok
11:26:41 DEBUG| [stderr] 12: DSO data reopen                            : FAILED!
11:26:41 DEBUG| [stderr] 13: Roundtrip evsel->name                      : Ok
11:26:41 DEBUG| [stderr] 14: Parse sched tracepoints fields             : Ok
11:26:41 DEBUG| [stderr] 15: syscalls:sys_enter_openat event fields     : Ok
11:26:41 DEBUG| [stderr] 16: Setup struct perf_event_attr               : FAILED!
11:26:41 DEBUG| [stderr] 17: Match and link multiple hists              : Ok
11:26:41 DEBUG| [stderr] 18: 'import perf' in python                    : Ok
11:26:41 DEBUG| [stderr] 22: Watchpoint                                 :
11:26:41 DEBUG| [stderr] 22.1: Read Only Watchpoint                     : FAILED!
11:26:41 DEBUG| [stderr] 22.2: Write Only Watchpoint                    : FAILED!
11:26:41 DEBUG| [stderr] 22.3: Read / Write Watchpoint                  : FAILED!
11:26:41 DEBUG| [stderr] 22.4: Modify Watchpoint                        : FAILED!
11:26:41 DEBUG| [stderr] 23: Number of exit events of a simple workload : Ok

```

Comment (narasimhan-v) [reply perf/perftool.py]: @PraveenPenguin so, the for loop is not needed here. You can parse it as a one whole string.

**Pull request merged**